<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestbedMainWindowClass</class>
 <widget class="QMainWindow" name="TestbedMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSelection">
    <property name="title">
     <string>Selection</string>
    </property>
    <addaction name="actionResetMask"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSelection"/>
  </widget>
  <widget class="QDockWidget" name="parametersWidget">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>209</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Parameters</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QComboBox" name="maskComboBox">
       <item>
        <property name="text">
         <string>Hue Mask</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Auto By Color</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Auto By Color(1)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="hue1SpinBox">
       <property name="maximum">
        <number>360</number>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="hue2SpinBox">
       <property name="maximum">
        <number>360</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Hue 1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Hue 2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Tollerance</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QSpinBox" name="toleranceSpinBox">
       <property name="maximum">
        <number>900</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="undoButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../resources/main.qrc">
         <normaloff>:/new/prefix1/back_alt.png</normaloff>:/new/prefix1/back_alt.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionResetMask"/>
   <addaction name="actionShowMask"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QDockWidget" name="logWidget_2">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Log</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../../../resources/main.qrc">
     <normaloff>:/new/prefix1/remove.png</normaloff>:/new/prefix1/remove.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionResetMask">
   <property name="icon">
    <iconset resource="../../../resources/main.qrc">
     <normaloff>:/new/prefix1/bullet_blue.png</normaloff>:/new/prefix1/bullet_blue.png</iconset>
   </property>
   <property name="text">
    <string>Reset Mask</string>
   </property>
  </action>
  <action name="actionShowMask">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../resources/main.qrc">
     <normaloff>:/new/prefix1/select.png</normaloff>:/new/prefix1/select.png</iconset>
   </property>
   <property name="text">
    <string>Show Mask</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../resources/main.qrc"/>
 </resources>
 <connections/>
</ui>
